### ISSUE TITLE
USWDS v3 Additional info - Design

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#1694]
## Description
USWDS doesn't have an Additional Info component so the work that is needed is to create an additional info v3 design that has updated colors, icon (if necessary) and typography to match the global USWDS style. 

 

## Details
[USWDS utilities](https://designsystem.digital.gov/utilities/)
[VA DS Additional info component](https://design.va.gov/components/additional-info) 

## Tasks
- [ ] Create designs for component
- [ ] Review designs with PO and/or DSC
- [ ] Address any comments from reviews, if necessary
- [ ] Comment on this ticket with any accessibility considerations engineers may need to know
- [ ] Comment on this ticket with content specifications (e.g. labels and error messages)
- [ ] Comment on this ticket with a link to the designs and post in DST Slack channel

## Acceptance Criteria
- [ ] Component design is complete and has been reviewed
- [ ] Accessibility considerations have been added to this ticket, if necessary
- [ ] Content specifications have been added to this ticket, if necessary
- [ ] Link to design has been added to this ticket and shared in Slack

**Comment by LillyBoot** [comment]
Keeping the current VA design for the MVP

**Comment by LillyBoot** [comment]
A couple notes about the design/future state:

1. Based on conversations during the weekly design meeting the MVP design will be the same as it currently is. 
2. The future V3 state will have the USWDS Icon instead of the font awesome
3. I created a mural board to compare the V3 info components. I added the potential redesign for the Additional Info component. https://app.mural.co/invitation/mural/test703663/1676046713199?sender=ucdde6a5c31c67de71bfb6448&key=4dadd5c3-e487-4609-8f7d-069f87617a65
